framework/core/src/Extend/ErrorHandling.php

```diff
@@ -76,7 +76,7 @@ public function type(string $exceptionClass, string $errorType): self
      * contain "details" - arbitrary data with more context for to the error.
      *
      * @param string $exceptionClass: The ::class attribute of the exception class.
-     * @param string $errorType: The ::class attribute of the handler class.
+     * @param string $handlerClass: The ::class attribute of the handler class.
```

Comment:
I believe that we can use `class-string<T>` for these. Although maybe best left for a followup PR so we can get this landed.

https://phpstan.org/blog/generics-by-examples

Reply:
Yup we can do a lot more with features like those other the higher we up the level of phpstan.